lightkurve/seismology/core.py

```diff
@@ -97,19 +110,11 @@ def _validate_deltanu(self, deltanu):
                 raise AttributeError("You need to call `Seismology.estimate_deltanu()` first.")
         return deltanu
 
-
-    def plot_echelle(self, deltanu=None, numax=None,
-                     minimum_frequency=None, maximum_frequency=None,
-                     smooth_filter_width=.1,
-                     scale='linear', cmap='Blues'):
-        """Plots an echelle diagram of the periodogram by stacking the
-        periodogram in slices of deltanu.
-
-        Modes of equal radial degree should appear approximately vertically aligned.
-        If no structure is present, you are likely dealing with a faulty deltanu
-        value or a low signal to noise case.
-
-        This method is adapted from work by Daniel Hey & Guy Davies.
+    def _clean_echelle(self, deltanu=None, numax=None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Seismology._clean_echelle is too complex (16)](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=4468966)